src/main/java/me/atam/atam4j/health/AcceptanceTestsHealthCheck.java

```diff
@@ -1,18 +1,22 @@
 package me.atam.atam4j.health;
 
 import com.codahale.metrics.health.HealthCheck;
-import org.junit.runner.notification.Failure;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Optional;
 
 public class AcceptanceTestsHealthCheck extends HealthCheck {
 
     public static final String TOO_EARLY_MESSAGE = "Too early to tell - tests not complete yet";
     public static final String OK_MESSAGE = "All is A OK!";
-    public static final String FAILURE_MESSAGE = "Number of failures:";
 
     private static AcceptanceTestsState testsState;
 
     public static final String NAME = "Acceptance Tests HealthCheck";
 
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Tried that, but getting escaped new line characters in input.
It might be good to actually expose the test report on a different REST endpoint so that a proper json is served, instead of an escaped version. 
And we can include brief description of number of failed tests and provide a link to that endpoint in this healthcheck message.
I'm not sure if the clients of `atam4j` would like a new endpoint being exposed in their app for this. Any thoughts @tjcutajar @anuragkapur @phillbarber